jsonenums.go

```diff
@@ -75,6 +75,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"./utils"
```

Comment:
i don't think we should import relative paths like this. I'm somewhat surprised it even works.